components/org.wso2.carbon.identity.api.server.idp/org.wso2.carbon.identity.api.server.idp.v1/src/gen/java/org/wso2/carbon/identity/api/server/idp/v1/IdentityProvidersApiService.java

```diff
@@ -20,6 +20,7 @@
 import org.wso2.carbon.identity.api.server.idp.v1.model.*;
 import org.apache.cxf.jaxrs.ext.multipart.Attachment;
 import org.apache.cxf.jaxrs.ext.multipart.Multipart;
+import java.io.InputStream;
```

Comment:
do we need this import?

Reply:
this is generated code, do we need to alter it?